HLTrigger/HLTanalyzers/test/test_hltrigreport_run_job.py

```diff
@@ -0,0 +1,4 @@
+from test_hltrigreport_base_cfg import process
+
+process.hlTrigReport.resetBy = "run"
+process.hlTrigReport.reportBy = "job"
```

Comment:
does it really make sense to reset every run and report only at the end of the job ?

Reply:
It would give you the last run of the job.
It was what the original code supported so I did the same.

Reply:
OK... IIRC the possibility was just an accident in the original code, not an intended feature.
Of course, having it doesn't harm, as long as it didn't require extra work to reimplement.

Reply:
> Of course, having it doesn't harm, as long as it didn't require extra work to reimplement.

It actually does require additional logic in order to implement. But it is there and working.